package.json

```diff
@@ -54,8 +54,6 @@
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.21.5",
     "@babel/register": "^7.4.0",
-    "@canvas-js/core": "0.5.0-alpha4",
-    "@hicommonwealth/eslint-plugin": "workspace:*",
```

Comment:
why did we remove `"@hicommonwealth/eslint-plugin": "workspace:*",`?

Reply:
Checked with Bob, looks like this happened during merging